php/environment.md

```diff
@@ -17,12 +17,12 @@ configuration file. These variables will be applied to all workers when they are
 server:
   command: "php worker.php"
   env:
-    DEBUG: true
+     APP_RUNTIME: prod
 ```
 
 {% endcode %}
 
-In this example, when RoadRunner starts a PHP worker, it will set the `DEBUG` environment variable to `true`.
+In this example, when RoadRunner starts a PHP worker, it will set the `APP_RUNTIME` environment variable to `prod`.
```

Comment:
Because may be meaning intersects with pool.debug=true

Reply:
Good. Could you please also merge the environment.md with the environment section in the [config plugin docs?](https://docs.roadrunner.dev/general/config#environment-variables)

Reply:
What does it mean: merge section? [config plugin docs?](https://docs.roadrunner.dev/general/config#environment-variables) already has link to `php/environment.md`. Should i doublicate section or ...?

Reply:
I mean - deduplicate. There are two places where the user can find information about env variables. So currently we have 2 sections: in the config plugin and in the worker section: https://docs.roadrunner.dev/php-worker/environment. And I'm considering merging the env section from the config plugin into the dedicated environmend.md in the php-worker.